roles/rhcos-post-install/tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   register: rhcos_region
 
 - name: Get public subnet name
-  shell: aws ec2 describe-subnets --filters "Name=tag:kubernetes.io/cluster/{{ OPENSHIFT_INSTALL_CLUSTER_NAME }},Values=owned" "Name=cidrBlock,Values=10.0.0.0/20" | grep "Name" | awk '{print $3}'
+  shell: aws ec2 describe-subnets --filters "Name=cidrBlock,Values=10.0.0.0/20" | grep "{{ OPENSHIFT_INSTALL_CLUSTER_NAME }}" | grep "Name" | awk '{print $3}'
```

Comment:
What about using the captured cluster name in the 1st task? (rhcos_cluster_name.stdout - note untested)  The grep has pluses and minuses, one bad example might be if people come very close to sharing a cluster name (Ex. akrzos-0 and akrzos-0-1 then the automation with akrzos-0 will capture two results right?)

Reply:
Makes sense to use rhcos_cluster_name.stdout instead of grep, update the PR @akrzos.